hedera-node/test-clients/src/main/resource/contract/contracts/CallInConstructor/CallInConstructor.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.8.0;
+
+contract CallInConstructor {
+    address constant CONSOLE_ADDRESS = address(0x000000000000000000636F6e736F6c652e6c6f67);
+
+    constructor() {
```

Comment:
Q: Can we expand the coverage with a method (not the constructor) doing the same in a follow up execution

Reply:
added